### ISSUE TITLE
Add Default Option Named Jira and Github

### ISSUE THREAD

**Issue by fr373969** [mshytikov/jira-shortcut#12]
Hello, I found this extension quite useful in communicating an issue with anyone, however when I tried to push this plug in to everyone, I found it a big challenge to setup the regex for my teammate, but it can be easily solved by adding the default option like below snippet under `config.js`

I understand that jira option below is almost exactly the same with existing markdown option, with the tweak for out_pattern, the reason I have the output like this is because our team use slack and I can just one click copy the title and url, formatted it, and store in the clipboard, and then paste into slack, and format the message into example like:

* JIRA: ![image](https://github.com/user-attachments/assets/c44e7a02-4437-483b-861b-dd6f648a084c) 
* GitHub: ![image](https://github.com/user-attachments/assets/3cebb7b5-944f-4a8d-a508-53f56d6f0655)

Provided these options right out of shell will be easier to enable people to use this great plugin.

```
    jira : {
      name: "JIRA",
      test_url : 'https://issues.apache.org/jira/browse/HADOOP-19092',
      test_title : '[HADOOP-19092] ABFS phase 4: post Hadoop 3.4.0 features - Enterprise JIRA',
      url_pattern : '(jira|tickets)*/browse/',
      title_pattern : '^\\[#?([^\\]]+)\\](.*)( -[^-]+)$',
      out_pattern : '`$1`: [$2]($url)'
    },
    github : {
      name: "GitHub",
      test_url : 'https://github.com/mshytikov/jira-shortcut/pull/10',
      test_title : 'Migrated to Manifest v3 by mshytikov · Pull Request #10 · mshytikov/jira-shortcut',
      url_pattern : 'github.*\\.com',
      title_pattern : '(.*) by .* · Pull Request (.*)',
      out_pattern : '*$1* $url'
    },
```

**Comment by mshytikov** [comment]
Hey @fr373969

It is a good idea, thank you for detailed suggestion.  
I will add this to default options this week. 

Will keep you posted.

**Comment by mshytikov** [comment]
Hey @fr373969 , I just released new version (v7.1.0) it is pending review and should be approved soon. 

After update you can find the buttons with your configuration  in the 'custom section' (I did very minor changes to proposed config) .
_Note:_ the buttons in the popup will have the name `Slack (v1)` but you know you can always edit it in the options settings.  
 
<img width="435" alt="Image" src="https://github.com/user-attachments/assets/5467fb74-2a94-4c14-85aa-d48530cb0478" />

**Comment by fr373969** [comment]
Awesome thank you so much

the name didn't change when clicking the button, but it is all good

![Image](https://github.com/user-attachments/assets/0d5905b5-ded1-4f83-b6f6-391b53f542e0)

**Comment by mshytikov** [comment]
yes fair it is not ideal - the name didn't change because this is the name which you see on the drop down,  and I decided to use the same name 'Slack(v1)' for this custom config. 
It is a bit of trade off of mapping button names to config setting. I could not come up with anything better at this point. :)  

Thank you for raising the issue and checking that it  works  👍

**Comment by fr373969** [comment]
Another feedback, our organization uses both public GitHub and enterprise
GitHub, thus the URL is will be
https://GitHub.company.com/organization/project

The GitHub pull request option was not working for it.

You've implemented beyond what I could imagine, and when I showcased the
function to that team and the leadership, they are all amazed by it, thank
you for making the huge organization communicate more efficiently and
effectively.

Many thanks
